src/clarity-angular/popover/dropdown/dropdown-toggle.ts

```diff
@@ -30,7 +30,7 @@ export class DropdownToggle {
     }
 
     get active(): boolean {
-        return this._dropdown.open;
+        return this._dropdown.ifOpenService.open;
     }
 
     @HostListener("click")
```

Comment:
This HostListener should just `ifOpenService.open = !ifOpenService.open`. It shouldn't delegate to the dropdown.